WatchGuard patches to sources for ssl_match_hostname-3.4.0.2 as of
Mon May  9 14:01:44 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- ssl_match_hostname-3.4.0.2/setup.py.orig	2022-05-09 14:01:43.952473914 -0700
+++ ssl_match_hostname-3.4.0.2/setup.py	2022-05-09 14:01:44.044470183 -0700
@@ -2,7 +2,10 @@
 # This setup.py was generated automatically by Pyron.
 # For details, see http://pypi.python.org/pypi/pyron/
 
-from setuptools import setup, find_packages
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
 
 setup(
     name = 'backports.ssl_match_hostname',
@@ -15,7 +18,7 @@
     classifiers = ['Development Status :: 5 - Production/Stable', 'License :: OSI Approved :: Python Software Foundation License', 'Programming Language :: Python :: 2.4', 'Programming Language :: Python :: 2.5', 'Programming Language :: Python :: 2.6', 'Programming Language :: Python :: 2.7', 'Programming Language :: Python :: 3', 'Programming Language :: Python :: 3.0', 'Programming Language :: Python :: 3.1', 'Topic :: Security :: Cryptography'],
 
     package_dir = {'': 'src'},
-    packages = find_packages('src'),
+    packages = ['backports', 'backports.ssl_match_hostname'],
     include_package_data = True,
     install_requires = [],
     entry_points = '',
